jest.config.js

```diff
@@ -1,6 +1,18 @@
 const { jestDirAlias } = require('./dirAlias');
 
 module.exports = {
+  reporters: [
+    'default',
+    [
+      'jest-junit',
+      {
+        suiteName: 'Jest Tests',
+        outputDirectory: 'reports',
```

Comment:
Maybe we should put these under `reports/jest` and then we can group the other results/reports under there too.

Reply:
- Yes, that's a good idea.
- Test reports are saved uniquely, so won't be overwritten until they're deleted.